crates/pathfinder/Cargo.toml

```diff
@@ -27,7 +26,10 @@ pedersen = { path = "../pedersen" }
 reqwest = { version = "0.11.4", features = ["json"] }
 rusqlite = { version = "0.26.1", features = ["bundled"] }
 serde = { version = "1.0.130", features = ["derive"] }
-serde_json = { version = "1.0.68", features = ["arbitrary_precision", "raw_value"] }
+serde_json = { version = "1.0.68", features = [
+    "arbitrary_precision",
+    "raw_value",
+] }
```

Comment:
Not sure this kind of formatting is really required but doesn't matter.

Reply:
Auto formatting on save kicked in, np.